src/composables/useSessionIssueHandler.js

```diff
@@ -48,7 +48,7 @@ export function useSessionIssueHandler() {
 				// FIXME: can't use router push as it somehow doesn't clean up
 				// fully and leads the other instance where "Join here" was clicked
 				// to redirect to "not found"
-				window.location = generateUrl(url)
+				window.location.replace(generateUrl(url))
```

Comment:
Could you describe this change?

Reply:
1. TS was complaining at the new value type
2. location.replace won't be saved in history, so user can't go back to the conflicting route
3. More clear from the method, that we want to trigger a page reload